evm/src/cpu/contextops.rs

```diff
@@ -96,7 +96,7 @@ fn eval_packed_get<P: PackedField>(
 
     // Unused channels.
     for i in 1..NUM_GP_CHANNELS {
-        if i != 3 {
+        if i != 2 {
```

Comment:
here I'd just remove the `if` and do the `for` loop with `for i in 3..NUM_GP_CHANNELS`, eventually adding a comment for clarity if needed

Reply:
alternatively I'm not even sure to understand why we constrain only channels 1 and 3 to be unused (range is end-exclusive). There are other channels not used no?

Reply:
0 is the top of the stack and may or may not be in use. 2 is not used by GET_CONTEXT, but it's used by SET_CONTEXT and must not be constrained.
All other channels are unused, which is only channel 1 now (since channel 3 doesn't exist anymore).